src/concepts/glean/glean.md

```diff
@@ -50,23 +73,12 @@ To make it easier to understand the general semantics of our data, the Glean SDK
   * `org_mozilla_fenix.baseline`
   * `org_mozilla_fenix.events`
   * `org_mozilla_fenix.metrics`
-  * Example query:
-
-    ```sql
-    -- Count unique Client IDs observed on a given day
-    SELECT
-      count(distinct client_info.client_id)
-    FROM
-      org_mozilla_fenix.baseline
-    WHERE
-      date(submission_timestamp) = '2019-11-11'
-    ```
   * There is [more documentation about accessing Glean data](accessing_glean_data.md).
 
 * _(Work in progress)_ Use events and [Amplitude](https://sso.mozilla.com/amplitude) for product analytics.
 * [Use Databricks](https://sso.mozilla.com/databricks) for deep-dive analysis.
 * [Use the Glean debug ping viewer](debug_ping_view.md) for QA & development.
-* For experimentation, you will be able to use [Android experiments library](https://github.com/mozilla-mobile/android-components/blob/master/components/service/experiments/README.md), which integrates with Glean.
+* For experimentation, you can use [Android experiments library](https://github.com/mozilla-mobile/android-components/blob/master/components/service/experiments/README.md), which integrates with Glean.
```

Comment:
@travis79: Does this still make sense as a link to Glean-adjacent experiments?  Should we like to Mako itself as well?

Reply:
I'm not sure I completely understand the question, but the link you provided dumps you right at Mako's README.md (it's still named `service-experiments` in A-C) and I don't think it's a bad thing to mention the integration between Mako and Glean.

Reply:
Ah, I guess I don't know the distinction between a-c's experiments and Mako.  Is that the same thing?  What's the backend part?  Is that Kinto?

Basically my question is -- if a user wants to add experiments to a Glean-using Android app, is this still the best link?

Reply:
a-c's experiments _is_ Mako, it was just never renamed.  The back-end part is a Mozilla hosted Kinto server.

To answer your question: yes, that is the best starting point for getting started with Mako experiments.

Reply:
To build on what @mdboom said: if Mako == `service-experiments`, then the README should clearly state that (Mako is not even mentioned once in [the docs](https://github.com/mozilla-mobile/android-components/tree/master/components/service/experiments)). Until that is rebranded to Mako, I think we should refrain from calling it Mako.. from our docs :)

Reply:
That's fine by me, rebranding it at the repo level was always going to be a chore anyway so "Mako" has always just been a sort of 'codename' anyway.